libc/src/__support/OSUtil/windows/exit.cpp

```diff
@@ -0,0 +1,20 @@
+//===----------- Windows implementation of an exit function -----*- C++ -*-===//
```

Comment:
Nit: should be:

```suggestion
//===-- Windows implementation of an exit function ------------------------===//
```

but there are existing files in OSUtil that don't conform to the standard file header either.

See https://llvm.org/docs/CodingStandards.html#file-headers.

Reply:
Updated.